api/src/main/resources/workbench.yaml

```diff
@@ -1517,6 +1517,20 @@ definitions:
         $ref: "#/definitions/Workspace"
         description: The newly created workspace clone.
 
+  UpdateWorkspaceRequest:
+    type: object
+    required:
+     - workspace
+    properties:
+      workspace:
+        $ref: "#/definitions/Workspace"
+
+  UpdateWorkspaceResponse:
```

Comment:
Sorry I was a bit ambiguous in my previous comment - I don't think a response wrapper is necessary here. For a REST update, I would always use the pattern of `UpdateWorkspaceRequest`, and a response of `Workspace`, per https://cloud.google.com/apis/design/standard_methods#update

One could argue that the clone response could also just be a Workspace, but my instinct is that since it's a non-standard REST method, I can't predict what kind of functionality it might need in the future and therefore there's a non-zero chance we might want to expand the response to include something else. I could probably be convinced to just return Workspace there though as well.